src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/ViewHelper.java

```diff
@@ -94,4 +105,37 @@ public static void set(
         setPivotXIfNeeded(view, pivotX);
         setPivotYIfNeeded(view, pivotY);
     }
+
+    @NonNull
+    public static LinearLayout createNavigationBarOuterLayout(Context context)
+    {
+        LinearLayout linearLayout = new LinearLayout(context);
+        linearLayout.setOrientation(LinearLayout.VERTICAL);
+        linearLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT));
+        return linearLayout;
+    }
+
+    @NonNull
+    public static FrameLayout createNavigationBarArea(Context context, LinearLayout linearLayout)
+    {
+        FrameLayout frameLayout = new FrameLayout(context);
+        frameLayout.setId(View.generateViewId());
```

Comment:
Since this view holds a fragment, it was crashing until I used an autogenerated id.